scripts/codegen-utils.js

```diff
@@ -0,0 +1,124 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+const packageJSON = require('../package.json');
+
+const ERROR_PREFIX = 'react-native-gesture-handler';
+const ROOT_DIR = path.resolve(__dirname, '..');
+const ANDROID_DIR = path.resolve(ROOT_DIR, 'android');
+const GENERATED_DIR = path.resolve(ANDROID_DIR, 'build/generated');
+const OLD_ARCH_DIR = path.resolve(ANDROID_DIR, 'src/paper');
+const SPECS_DIR = path.resolve(ROOT_DIR, packageJSON.codegenConfig.jsSrcsDir);
+const PACKAGE_NAME = packageJSON.codegenConfig.android.javaPackageName;
+const RN_DIR = path.resolve(ROOT_DIR, 'node_modules/react-native');
+const RN_CODEGEN_DIR = path.resolve(
+  ROOT_DIR,
+  'node_modules/@react-native/codegen'
+);
+
+const SOURCE_FOLDERS = 'java/com/facebook/react/viewmanagers';
```

Comment:
add the other mentioned dir here too.

Reply:
fixed in: https://github.com/software-mansion/react-native-svg/pull/2357/commits/15900e9c96691ac87f64610e9eeffce09c6135c8